Source/Core/AudioCommon/WaveFile.h

```diff
@@ -30,7 +30,7 @@ class WaveFileWriter : NonCopyable
   void Stop();
 
   void SetSkipSilence(bool skip) { skip_silence = skip; }
-  void AddStereoSamplesBE(const short* sample_data, u32 count, int sample_rate);  // big endian
+  void AddStereoSamples(const short* sample_data, u32 count, int sample_rate, bool big_endian);
```

Comment:
Generally prefer enums to boolean flags like that. If I look at a call site and I see "true" it doesn't tell me what that means. LITTLE_ENDIAN / BIG_ENDIAN would be clearer.